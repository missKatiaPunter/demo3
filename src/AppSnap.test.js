import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

test("that DOM renders consistently", () => {
    const tree = renderer.create(<App/>).toJSON();
    expect(tree).toMatchSnapshot();
})