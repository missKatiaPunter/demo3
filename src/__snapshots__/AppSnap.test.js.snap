// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`that DOM renders consistently 1`] = `
<div
  className="App"
>
  <p>
    Hello World!
  </p>
</div>
`;
